### PR TITLE
🚧✨[PROD-1636] Update orders API docs

### PR DESCRIPTION
**Pull request by knor-el-snor**
Wait until https://github.com/ShopBonsai/bonsai/pull/15397 gets deployed to production!

### CODE REVIEW

File: docs/api/orders/submit-orders.api.mdx [hunk 1]
```diff
@@ -5,7 +5,7 @@ description: "Submits an order to Bonsai for processing. Be careful as these ord
 sidebar_label: "Submit Orders"
```
Comment (knor-el-snor) [inline docs/api/orders/submit-orders.api.mdx:5]: Not sure why this file changed? Did we forget to add anything last time? It's really annoying to see the git diff if these are unformatted

File: docs/api/orders/order-status-by-id.api.mdx [hunk 2]
```diff
@@ -31,7 +31,7 @@ Shows information about an existing order and its products by Bonsai order id. T
...
...={"{\n  \"data\": {\n    \"id\": \"cldbvy7gt0006hpzo9nyw72ks\",\n    \"externalId\": \"51cf91c1-da4a-42c4-81d3-3ec3e3268655\",\n    \"fulfillmentStatus\": \"fulfilled\",\n    \"products\": [\n      {\n        \"id\": \"clad6pg5z00eu012gfz7hfm92\",\n        \"variantId\": \"M00679529706740\",\n        \"quantity\": 1,\n        \"merchantOrderId\": \"T8745295\",\n        \"fulfillmentStatus\": \"fulfilled\",\n        \"paymentStatus\": \"paid\"\n      }\n    ],\n    \"shippingTracking\": [\n      {\n        \"carrier\": \"USPS\",\n        \"trackingNumber\": \"9400100000000000069420\",\n        \"trackingUrl\": \"https://tools.usps.com/go/TrackConfirmAction_input?qtc_tLabels1=9400100000000000069420\",\n        \"products\": [\n          {\n            \"id\": \"clad6pg5z00eu012gfz7hfm92\",\n            \"variantId\": \"M00679529706740\",\n            \"quantity\": 1\n          }\n        ]\n      }\n    ],\n    \"customer\": {\n      \"email\": \"qa@shopbonsai.ca\",\n      \"firstName\": \"string\",\n      \"lastName\": \"string\"\n    }\n  }\n}"} language={"json"}></ResponseSamples></TabItem></SchemaTabs></TabItem></MimeTabs></div></TabItem><TabItem label={"400"} value={"400"}><div>
```
Comment (knor-el-snor) [inline docs/api/orders/order-status-by-id.api.mdx:34]: Main change is here in the return type for this API call

Comment (knor-el-snor) [inline docs/api/orders/order-status-by-id.api.mdx:34]: Main change is here in the return type for this API call

**Review by KameronSturgess — APPROVED** [review]
git diff is illegible so I'm trusting in the auto generation. For submit-orders I missed updating it to include 'paymentDescription' so that's likely the change

**Comment by alenkis** [comment]
@knor-el-snor is this PR still relevant?

**Comment by knor-el-snor** [comment]
@alenkis Given you deployed more recent docs, nope! Will close it

**Pull request closed without merging**